source/manual/republishing-content.html.md.erb

```diff
@@ -21,35 +25,65 @@ If the documents are in Whitehall, there are Rake tasks you can run as outlined
+
+### To republish all documents of a specific type
+
+To republish all instances of the following document types, run the following rake task.
+
+- CaseStudy
+- Consultation
+- Contact
+- CorporateInformationPage
+- DetailedGuide
+- DocumentCollection
+- FatalityNotice
+- Government
+- NewsArticle
+- OperationalField
+- Organisation
+- Person
+- PolicyGroup
+- Publication
+- Role
+- RoleAppointment
+- Speech
+- StatisticalDataSet
+- StatisticsAnnouncement
+- TakePartPage
+- TopicalEvent
+- TopicalEventAboutPage
+- WorldLocation
+- WorldwideOrganisation
 
 <%= RunRakeTask.links("whitehall", "publishing_api:bulk_republish:document_type[DocumentClass]") %>
 
-For example:
-`/government/case-studies/alexander-dennis-maximum-capacity`
-`publishing_api:bulk_republish:document_type[CaseStudies]`
+> Replace `DocumentClass` with the camelized (i.e. as it is written above) class name.
 
-You may wish to test first on Integration.
+### To republish multiple documents
 
-For a short list of Content IDs, use the `documents_by_content_ids` rake task:
+For a small number of documents, use the following rake task:
 
 <%= RunRakeTask.links("whitehall", "publishing_api:bulk_republish:documents_by_content_ids[content_id_1 content_id_2]") %>
 
-For a significant number of Content IDs, some preparation is needed for this as a CSV file needs to be in place.
+> Replace `content_id_1` and `content_id_2` with the content ID (i.e. UUID) for the documents to republish. You can add more than 2 content IDs.
+
+For a significant number of documents, a CSV file should be added to the repository:
 
-1. The CSV should have a column called content_id that contains all the relevant IDS. This should be added to the whitehall repository at `lib/tasks/{FILENAME}.csv`.
+1. Create a CSV file that contains a single column headed `content_id`. Put the content ID for each document on a separate line below this. The file should be saved in `lib/tasks/{FILENAME}.csv` and a PR raised.
+1. Merge and deploy the PR to the relevant environment.
 1. Run the `documents_by_content_ids_from_csv` rake task:
    <%= RunRakeTask.links("whitehall", "publishing_api:bulk_republish:documents_by_content_ids_from_csv[csv_file_name]") %>
+   > Replace `csv_file_name` with the filename of the CSV, including the `.csv` extension.
 1. After the job has completed, remove the CSV from the repository.
 
-To republish all documents:
-Caution: this is a lot of content and will take hours to complete. If it is possible to scope the republish do so and use a different task, but if you have made a change such as something in govspeak that will affect the majority of content, this is available. Before running this job confirm with Technical 2nd Line that they are happy for you to proceed as it could cause backed up publishing queues and alerts.
+#### To republish all documents
```

Comment:
Minor - should this be an h3 rather than h4 for consistency with above section?

Reply:
Yes, it should be.  I'll fix this.